source/releasenotes/2025-07-08-tls-10-11-eol.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "TLS 1.2 or 1.3 required for external connections starting August 18, 2025"
+published_date: "2025-07-08"
+categories: [action-required, infrastructure, security]
+---
+
+Starting August 18, 2025, Pantheon will begin rolling out [PHP Runtime Generation 2](/php-runtime-generation-2) to all sites on the platform. The new PHP Runtime enforces the use of TLS 1.2 or 1.3 for external connections. This change is part of our ongoing efforts to enhance security and comply with modern best practices.
+
+## What is TLS?
+
+Transport Layer Security (TLS) is the cryptographic protocol that secures connections between systems — for example, when your website connects to a third-party API. TLS 1.0 and 1.1 were introduced in 1999 and 2006, respectively, and are now considered deprecated due to known vulnerabilities and lack of support for modern encryption standards.
+
+TLS 1.2 was released in 2008 and remains the industry standard, with widespread adoption across browsers and APIs. TLS 1.3, released in 2018, offers additional performance and security benefits.
+
+## Action Required
+
+Sites using external services that still rely on TLS 1.0 or 1.1 will experience failed connections once PHP Runtime Generation 2 is enabled. We strongly recommend auditing your site’s outgoing connections to ensure compatibility with TLS 1.2 or higher.
+
+Follow our [TLS testing guide](/guides/tls-testing) to check whether your site is making external requests using outdated TLS versions. Our DevRel team has created [CMS modules](/tls-compatibility#how-do-i-check-tls-compatibility-for-my-site) that can help expedite your testing.
```

Comment:
"TLS testing guide" does not exist, so that link will 404.
```suggestion
Follow our [guide for testing TLS compatibility](/tls-compatibility#how-do-i-check-tls-compatibility-for-my-site) to check whether your site is making external requests using outdated TLS versions. Our DevRel team has created a [WordPress plugin](https://github.com/pantheon-systems/wp-tls-compatibility-checker) and [Drupal module](https://github.com/pantheon-systems/drupal_tls_checker) that can help expedite your testing.
```